war/src/test/java/io/lumeer/engine/controller/SecurityFacadeIntegrationTest.java

```diff
@@ -106,8 +108,27 @@ public void addAndRemoveOrganizationUserRole() throws Exception {
       securityFacade.addOrganizationUserRole(org, user, roleManage);
       assertThat(securityFacade.hasOrganizationRole(org, roleManage)).isTrue();
 
+      List<Role> roles = securityFacade.getOrganizationRoles(org);
+      assertThat(roles).extracting(role -> role.getName())
+                       .containsOnly(LumeerConst.Security.RESOURCE_ROLES.get(
+                             LumeerConst.Security.ORGANIZATION_RESOURCE).toArray(new String[LumeerConst.Security.RESOURCE_ROLES.get(
+                                   LumeerConst.Security.ORGANIZATION_RESOURCE).size()]));
+
+      assertThat(roles).filteredOn(role -> role.getName().equals(roleManage))
+                       .extracting(role -> role.getUsers().contains(user));
```

Comment:
`extracting()` method itself does not check anything. You need to call `extracting(role -> role.getUsers()).contains(user)`.